src/client/src/routes/SimpleLauncher/tools.ts

```diff
@@ -85,3 +86,31 @@ export function createOpenFinWindow({ name, url, provider: { options } }: Config
     )
   })
 }
+
```

Comment:
feels to me like single responsibility violation and `tools.ts` is starting to do too much (arguably was before this addition).  I'd consider at this point splitting out these handlers?

Reply:
I am going to leave it as is and will address this comment in the subsequent PR.